cookiecutter/{{cookiecutter.github_repo_name}}/mkdocs.yml

```diff
@@ -121,8 +121,8 @@ plugins:
             # See https://mkdocstrings.github.io/python/usage/#import for details
             - https://docs.python.org/3/objects.inv
 {%- if cookiecutter.type in ("actor", "app", "model") %}
-            - https://frequenz-floss.github.io/frequenz-channels-python/v0.16/objects.inv
-            - https://frequenz-floss.github.io/frequenz-sdk-python/v0.25/objects.inv
+            - https://frequenz-floss.github.io/frequenz-channels-python/v1/objects.inv
+            - https://frequenz-floss.github.io/frequenz-sdk-python/v1.0-pre/objects.inv
```

Comment:
901 sounds safer, no?  esp. because we're limiting to 9xx in pyproject.toml

Reply:
There is no `901`, I just thought it was a better balance to only save the latest patch version for every minor version to make the versions menu a bit easier to navigate, as it is pretty basic. This is just another *casualty* of the eternal rc state, if it wouldn't be for that we should be fine with only serving the latest patch for every minor version. :shrug:

Reply:
I don't know what you mean by there's no 901, because that's literally the subject of the PR, but I may be missing something.  But it sounds like you know what you're doing.  So I'll approve.

Reply:
I mean in the generated docs, and because of that, in the index files used to do docs cross-referencing. These are the available documentation versions we can cross-link to: https://frequenz-floss.github.io/frequenz-sdk-python/versions.json.

Reply:
Oh, I see, so there are no docs for the older prereleases.